src/actions/analytics.ts

```diff
@@ -70,7 +71,8 @@ export const trackScreenChange = (
 ) => {
   const { analytics } = getState();
   const { [ANALYTICS_MCID]: mcid } = analytics;
-
+  // If in debug mode, set user property 'debug' to true, if prod set to false
+  FBAnalytics().setUserProperties({ debug: __DEV__ ? 'true' : 'false' });
```

Comment:
I'm not sure if this comment is helpful. Also debug is expecting strings and won't take a boolean?
```suggestion
  FBAnalytics().setUserProperties({ debug: __DEV__.toString() });
```

Reply:
I'll remove the comment since the logic is pretty self explanitory. And yeah it has to be a string.